lib/fog/vsphere/requests/compute/vm_clone.rb

```diff
@@ -82,8 +82,8 @@ def vm_clone(options = {})
 
           # Options['dest_folder']<~String>
           # Grab the destination folder object if it exists else use cloned mach
-          dest_folder = get_raw_vmfolder(options['dest_folder'], options['datacenter']) if options.key?('dest_folder')
-          dest_folder ||= vm_mob_ref.parent
+          dest_folder_path = options.fetch('dest_folder','/') # default to root path ({dc_name}/vm/)
```

Comment:
It is actually that I put my templates in a "templates" folder and I don't want my vm's to be cloned into that folder.
In the line below, I found that the `convert_vm_mod_ref_to_attr_hash` was needed because `get_virtual_machine` would fail to find the vm.